src/parameters/parameters_flow_solver.h

```diff
@@ -40,16 +40,29 @@ class FlowSolverParam
      *  will read file: input_mesh_filename.msh */
     std::string input_mesh_filename;
 
-    /** For integration test purposes, expected kinetic energy at final time. */
-    double expected_kinetic_energy_at_final_time;
-
     bool restart_computation_from_file; ///< Restart computation from restart file
     bool output_restart_files; ///< Output the restart files
     std::string restart_files_directory_name; ///< Name of directory for writing and reading restart files
     int restart_file_index; ///< Index of desired restart file for restarting the computation from
     int output_restart_files_every_x_steps; ///< Outputs the restart files every x steps
     double output_restart_files_every_dt_time_intervals; ///< Outputs the restart files at time intervals of dt
 
+    /** For taylor green vortex integration tests, expected kinetic energy at final time. */
```

Comment:
Feels like a lot of the current variables can be protected or private
public: accessed by other classes
protected: may be used by derived classes
private: only used for this specific class

Reply:
how would i access them outside this class if they're protected or private?